KeyboardHelperTests/Tests/KeyboardHelperTests.swift

```diff
@@ -9,6 +9,93 @@
 import XCTest
 @testable import KeyboardHelper
 
+class ShowSpyDelegate : KeyboardNotificationDelegate {
```

Comment:
Not a super fan of the names :D Could have been something more like `KeyboardShowingObject` and `KeyboardHidingObject`, but on the other hand, it's not a big issue.